Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM eclipse-temurin:17-jdk-jammy as builder
+
+RUN apt update && apt-get install maven -y
+
+WORKDIR /app
+
+RUN keytool -genkey -alias sitename -keyalg RSA -keystore keystore.jks -keysize 2048 -storepass 123456 -dname "CN=DD, OU=DD, O=DD, L=DD, S=DD, C=DD"
+
+COPY pom.xml ./
+COPY src/ ./src
+
+RUN mvn clean install
+
+FROM eclipse-temurin:17-jdk-jammy
```

Comment:
Let's add a Dockerfile in a separate PR.  But I am concerned with using such an old JDK instead of something more recent and streamlined like S3Proxy does:

https://github.com/gaul/s3proxy/blob/master/Dockerfile